.gitlab-ci.yml

```diff
@@ -1,9 +1,8 @@
-image: docker:latest
-
-services:
-  - docker:dind
-
 build and push emulator image:
```

Comment:
A bit of scope of your PR but I am not very sure this job is being used in any way. Maybe @prusnak knows?

Reply:
Ok thanks for feedback, aded description and as for the commits i will use gitlab: something something

Reply:
no idea

Reply:
@vdovhanych let's remove it, I do not think it is being used at this moment.